ledger-core/v2/virtual/handlers/handle_meta.go

```diff
@@ -104,6 +103,16 @@ func FactoryMeta(message *statemachine.DispatcherMessage) (pulse.Number, smachin
 			ctx.SetTracerID(traceID)
 			return &SMVStateUnavailable{Meta: payloadMeta, Payload: &pl}
 		}
+	case payload.TypeVDelegatedRequestFinished:
+		pl := payload.VDelegatedRequestFinished{}
```

Comment:
@ruz @bigbes @sexyhamster 
btw there is no reason to use pointers for meta and messages:
1. Copy is cheap
2. SM can be created first, and then a message is unmarshalled into the field